src/Agent/NewRelic/Agent/Core/Configuration/ServerConfiguration.cs

```diff
@@ -211,7 +214,7 @@ public class AgentConfig
             public IEnumerable<KeyValuePair<string, IEnumerable<string>>> ErrorCollectorExpectedMessages { get; set; }
 
             [JsonProperty("error_collector.expected_status_codes")]
-            public string ErrorCollectorExpectedStatusCodes { get; set; }
+            public IEnumerable<object> ErrorCollectorExpectedStatusCodes { get; set; }
```

Comment:
This should be a string.

Reply:
The server response value for this is actually an array of integers. I am using object here instead of int because there is a possibility that server response value would be be an array of strings when server side config for this starts to support a range of status codes.